src/templates/profile.ts

```diff
@@ -10,8 +13,104 @@ version = "${version}"
 export function empty(name: string): string {
   return `
 """
-${name} usecase
+${name} use case
+Use case description
 """
 usecase ${name} {}
 `;
 }
+export function completeProfile({
+  scope,
+  name,
+  version,
+  usecaseNames,
+}: {
+  name: string;
+  scope?: string;
+  version: string;
+  usecaseNames: string[];
+}): string {
+  const id = ProfileId.fromScopeName(scope, name);
+  const formattedName = startCase(id.id);
+
+  const error = errorModel(scope);
+
+  return [
+    `"""
+${formattedName}
+TODO: What "${formattedName}" does
+"""
+name = "${id.id}"
+version = "${version}"
+// Comlink Profile specification: https://superface.ai/docs/comlink/profile`,
+    ...usecaseNames.map(u => usecase(u, error.statement)),
+    error.namedModel,
+  ].join('\n');
+}
+
+function usecase(useCaseName: string, errorStatement: string): string {
+  const formattedName = startCase(useCaseName);
+
+  return `
+"""
+${formattedName}
+TODO: What "${formattedName}" does
+"""
+usecase ${useCaseName} unsafe { // change safety to \`safe\`, \`idempotent\` or \`unsafe\`
+  input {
+    "Foo title"
+    foo! string 
```

Comment:
Btw I feel like we don't use non-nullable types often enough. Is it something worth teaching in the template?

```suggestion
    foo! string!
```